src/SDDPoptimize.jl

```diff
@@ -135,25 +135,29 @@ function run_SDDP!(model::SPModel,
         callsolver_backward = backward_pass!(model, param, V, problems, stockTrajectories, model.noises)
 
         ####################
+        # Update stats
+        lwb = get_bellman_value(model, param, 1, V[1], model.initialState)
+        updateSDDPStat!(stats, callsolver_forward+callsolver_backward, lwb, upb, toq())
+        print_current_stats(stats,verbose)
+
+        ####################
         # cut pruning
-        prune_cuts!(model,param,V,stats.niterations,verbose)
+        if param.pruning[:pruning]
```

Comment:
This is not straightforward as we need to redefine problems object after some iterations. And if we do not want to do some cuts pruning, I prefer to avoid to call an unnecessary function (that's better for memory allocation).